### ISSUE TITLE
How to load pretained weights?

### ISSUE THREAD

**Issue by junhyukso** [google-research/rigl#7]
Hi!
How can I load pertained weights in README?

I want to extract those weight for research purpose.

Can you tell me how can I load it? or can give me brief code to make model and load weights?

**Comment by evcu** [comment]
I added section to evaluate models [here](https://github.com/google-research/rigl#evaluating-checkpoints).

```python
reader = tf.train.load_checkpoint(ckpt_path)
for name in tf.train.list_variables(ckpt_path):
 weights = reader.get_tensor(name)
 print(name, weights.shape)
```

Something like this should work for reading weights/masks.
